test/unit/traversing.js

```diff
@@ -743,6 +743,37 @@ QUnit.test( "contents()", function( assert ) {
 	assert.equal( c[ 0 ].nodeValue, "hi", "Check node,textnode,comment contents is just the one from span" );
 } );
 
+QUnit.test( "contents() for <template />", function( assert ) {
```

Comment:
I confused about it is necessary to `assert.ok()` when I use `QUnit.skip()`? Or it's just a placeholder like [this example](http://api.qunitjs.com/QUnit.skip/#example-0)?